src/store/types/index.ts

```diff
@@ -353,6 +371,38 @@ export interface IRole {
     __v: number;
 }
 
+export interface IAssignSecondaryRole {
+    departments: {
+        campusId: string;
+        departmentId: string;
+    }[];
+    email?: string;
+    userId: string;
+    roleId: string;
+}
+
+// Department
+export interface IDepartment {
+    _id: string;
+    departmentName: string;
+    campusId: string;
+    description: string;
+    createdAt: string;
+    __v: number;
+}
```

Comment:
I know there is already an interface called "IDepartment". Why not use that or create "IGHDepartment" by picking fields from "IDepartment".
Eg:

`export type IGHDepartment = Pick<IDepartment, '_id' | 'departmentName' | 'campusId' | 'description' | 'createdAt'>;`